lib/services/service_brokers/v2/client.rb

```diff
@@ -92,11 +92,15 @@ def fetch_service_instance_state(instance)
 
     def bind(binding)
       path = service_binding_resource_path(binding)
-      response = @http_client.put(path, {
-        service_id:  binding.service.broker_provided_id,
-        plan_id:     binding.service_plan.broker_provided_id,
-        app_guid:    binding.app_guid
-      })
+      attr = {
+          service_id:  binding.service.broker_provided_id,
+          plan_id:     binding.service_plan.broker_provided_id
+      }
+      if binding.respond_to? 'app_guid'
```

Comment:
yes, we missed to test this piece of code in client spec, add one more test case in the latest patch